common/app/conf/switches/CommercialSwitches.scala

```diff
@@ -201,7 +201,7 @@ trait CommercialSwitches {
     description = "Turn on Sonobi header bidding",
     owners = Seq(Owner.withGithub("rich-nguyen"), Owner.withGithub("janua")),
     safeState = Off,
-    sellByDate = new LocalDate(2017, 3, 31), //Friday
+    sellByDate = new LocalDate(2017, 4, 28), //Friday
```

Comment:
Do these tests need to be extended for that long? Is it more sensible to extend them just until next week?